gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/testing/LocalResourceManagerHelper.java

```diff
@@ -324,12 +322,9 @@ Response create(Project project) {
     }
   }
 
-  Response delete(String projectId) {
+  synchronized Response delete(String projectId) {
```

Comment:
At the moment list (and maybe get in the future) don't have a filter that applies on life-cycle.
If that will change you may have a problem where the output and the check may not match (as you change the same object). If you want you can avoid it by always replacing the Project (like in replace method).